index.html

```diff
@@ -32,130 +34,144 @@ <h2 class="first-sheet__subtitle">LET’S CONNECT</h2>
+      <h2 class="first-sheet__subtitle">LET’S CONNECT</h2>
+      <ul class="sixth-sheet__icons">
+        <li><a href="#"><img src="./icons/github.svg" alt="github"></a></li>
+        <li><a href="#"><img src="./icons/linkdin.svg" alt="linkedIn"></a></li>
+        <li><a href="#"><img src="./icons/rabbit.svg" alt="rabbit"></a></li>
+        <li><a href="#"><img src="./icons/twitter.svg" alt="twitter"></a></li>
+        <li><a href="#"><img src="./icons/media.svg" alt="media"></a></li>
+      </ul>
+      <a href="#" class="sixth-sheet__button sixth-sheet__text">Get my resume</a>
+    </article>
+    <div class="sixth-sheet__developer">
+      <div class="sixth-sheet__developer-language">
         <h3 class="sixth-sheet-developer-text">Languages</h3>
         <img class="sixth-sheet__developer-language-union" src="./images/union.png" alt="connection_link">
-      </li>
-      <li class="sixth-sheet__developer-language-boxes">
-          <img class="sixht-sheet__developer-language-img" src="./icons/javascript.png" alt="javascript">
-          <span>JavaScript</span>
-      </li>
-      <li class="sixth-sheet__developer-language-boxes">
-        <img class="sixht-sheet__developer-language-img" src="./icons/html.png" alt="html">
-        <span>HTML</span>
-      </li>
-        <li class="sixth-sheet__developer-language-boxes">
-          <img class="sixht-sheet__developer-language-img" src="./icons/css.png" alt="css">
-          <span>CSS</span>
-      </li>
-      <li class="sixth-sheet__developer-language">
+      </div>
+      <div class="sixth-sheet-languages">
+        <div class="sixth-sheet__developer-language-boxes">
+            <img class="sixht-sheet__developer-language-img" src="./icons/javascript.png" alt="javascript">
+            <span>JavaScript</span>
+        </div>
+        <div class="sixth-sheet__developer-language-boxes">
+          <img class="sixht-sheet__developer-language-img" src="./icons/html.png" alt="html">
+          <span>HTML</span>
+        </div>
+          <div  class="sixth-sheet__developer-language-boxes">
+            <img class="sixht-sheet__developer-language-img" src="./icons/css.png" alt="css">
+            <span>CSS</span>
+        </div>
+      </div>
+      <div class="sixth-sheet__developer-language">
         <h3 class="sixth-sheet-developer-text">Frameworks</h3>
         <img class="sixth-sheet__developer-language-union" src="./images/union.png" alt="connection_link">
-      </li>
-      <li class="sixth-sheet__developer-language">
+      </div>
+      <div class="sixth-sheet__developer-language">
         <h3 class="sixth-sheet-developer-text">Skills</h3>
         <img class="sixth-sheet__developer-language-union" src="./images/union.png" alt="connection_link">
-      </li>
-    </ul>
+      </div>
+    </div>
   </section>
   <section class="seventh-sheet">
```

Comment:
Figma Design
![Screenshot from 2022-04-30 21-42-40](https://user-images.githubusercontent.com/77842237/166118527-35f8f4b6-0c68-4842-be8d-4119213dbeb1.png)

Your Design
![Screenshot from 2022-04-30 21-43-16](https://user-images.githubusercontent.com/77842237/166118543-780b6059-f5e7-4b5b-8c57-6e9ab8e10a4a.png)



- [ ] Kindly You must replicate the design as much as possible (e.g., font, colors, images, tests, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).